federated_learning/financial_fraud_detection/README.md

```diff
@@ -80,6 +88,22 @@ cd tree-based/
 nvflare simulator jobs/creditcard_5_bagging_IID_split_uniform_lr -w ${PWD}/workspaces/xgboost_workspace_5_bagging_exponential_split_scaled_lr -n 5 -t 5
 ```
 
+You will see the execution as shown in the picture.
+![execution](./figures/execution.png)
+
+Each client loads the global model and starts local training. The application will periodically print the train-auc (AUC calculated from the training dataset) and valid-auc (AUC calculated from the validation dataset).
+After the federated learning phase finishes, the server will store the global model.
+![execution](./figures/store_model.png)
+
+#### Step 4. Prediction by the global model
+
+This is a simple example that utilizes a specific piece of data from the dataset to confirm that the prediction result is consistent with the ground truth. Additionally, you have the flexibility to modify the prediction inputs yourself based on your needs.
+![execution](./figures/GM_predict.png)
+
+```
+python GM_predict.py
```

Comment:
To ensure the user moves to the correct directory, I suggest adding `cd ..` before this command.

Reply:
done